SpeakGoodBye.js

```diff
@@ -0,0 +1,12 @@
+(function (window) {
+  var byeSpeaker = {};
+  var speakWord = "Good Bye";
```

Comment:
Use `const` for constants: https://www.w3schools.com/js/js_const.asp

Reply:
Noted, thank you